diff --git a/node_modules/highcharts-rounded-corners/rounded-corners.js b/node_modules/highcharts-rounded-corners/rounded-corners.js
index 365c2f4..19f2061 100644
--- a/node_modules/highcharts-rounded-corners/rounded-corners.js
+++ b/node_modules/highcharts-rounded-corners/rounded-corners.js
@@ -35,6 +35,12 @@
                 rTopRight = rel(options.borderRadiusTopRight || 0, w),
                 rBottomRight = rel(options.borderRadiusBottomRight || 0, w),
                 rBottomLeft = rel(options.borderRadiusBottomLeft || 0, w);
+            if (point.y < 0) {
+                rTopLeft = rel(options.borderRadiusBottomLeft || 0, w);
+                rTopRight = rel(options.borderRadiusBottomRight || 0, w);
+                rBottomRight = rel(options.borderRadiusTopLeft || 0, w);
+                rBottomLeft = rel(options.borderRadiusTopRight || 0, w);
+            }
 
             if (rTopLeft || rTopRight || rBottomRight || rBottomLeft) {
                 var maxR = Math.min(w, h) / 2
